test/realtime/socket-events.test.js

```diff
@@ -570,13 +581,80 @@ describe('realtime#socket event', function () {
       }, 5)
     })
 
-    it('should change permission to freely when config allowAnonymousEdits is true', function (done) {
+    it('should not change permission to freely when config allowAnonymousEdits is true', function (done) {
```

Comment:
Since `allowAnonymousViews` is `true` by default, there is no impact on the current user.